metatensor-core/tests/cmake-project/CMakeLists.txt

```
@@ -6,6 +6,12 @@ project(metatensor-test-cmake-project C CXX)
 
 option(USE_CMAKE_SUBDIRECTORY OFF)
 
+if (MINGW)
+    # CI can't find libsdc++, so we statically link it
```

Not only CI but always when trying to compile locally on 🪟 .

```suggestion
    # can't find libsdc++, so we statically link it
```